server/db/migrate/20150123165155_create_deployments.rb

```diff
@@ -4,9 +4,21 @@ def change
       t.string :name
       t.integer :lifecycle_environment_id
       t.integer :organization_id
```

Comment:
should we add ":null => false" to require any of the fields?  e.g. name, lifecyle_environment_id, organization_id